csvqb/csvqb/writers/qbwriter.py

```diff
@@ -421,3 +422,25 @@ def _get_measure_uri(measure: QbMeasure) -> str:
         return _doc_rel_uri(f"measure/{measure.uri_safe_identifier}")
     else:
         raise Exception(f"Unmatched unit type {type(unit)}")
+
+
+def _get_about_url(cube: Cube) -> str:
+    # Todo: Dimensions are currently appended in the order in which the appear in the cube.
+    #       We may want to alter this in the future so that the ordering is from
+    #       least entropic dimension -> most entropic.
+    #       E.g. http://base-uri/observations/male/1996/all-males-1996
+    multi_measure_cols = []
+    aboutUrl = "#obs"
+    for c in cube.columns:
+        if isinstance(c, QbColumn):
+            c: QbColumn
```

Comment:
I think this is [breaking the build](https://ci.floop.org.uk/job/GSS_data/job/csvwlib/view/change-requests/job/PR-98/1/console).

> /var/jenkins_home/workspace/GSS_data_csvwlib_PR-98/csvqb/csvqb/writers/qbwriter.py:434:9 - error: Expression of type "CsvColumn" cannot be assigned to declared type "QbColumn[Unknown]"